### ISSUE TITLE
after downloading recent version,run train.py.But get train loss nan and val map 0 why?is it related to cuda11.0 version?

### ISSUE THREAD

**Issue by jiangjiaxiaotianxia** [ultralytics/yolov5#4337]
I use gtx 1660ti ,my torch1.7+cuda11.0 ,during from the first epoch the train and val loss is nan ,map=0,and it  stays that way!
before train start ,there is a warining below:
** UserWarning: Detected call of lr_scheduler.step() before optimizer.step(). In PyTorch 1.1.0 and later, you should call them in the opposite order: optimizer.step() before lr_scheduler.step(). Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.**

the above is the question ,I don't know why , can anyone help ?Thanks!

**Comment by sunzhuojun** [comment]
I use gtx 3090,my torch1.8.1+cuda11.2,I trained my own data with new train.py, all losses I got were nan and mAP=0.
[results.xls](https://github.com/ultralytics/yolov5/files/6965193/results.xls)

**Comment by Rickyyy-zh** [comment]
> I use gtx 3090,my torch1.8.1+cuda11.2,I trained my own data with new train.py, all losses I got were nan and mAP=0.
> [results.xls](https://github.com/ultralytics/yolov5/files/6965193/results.xls)

Hi, have you solve it?

**Comment by richard880502** [comment]
try  to  use cuda10.2

**Comment by zzff-sys** [comment]
Hello, I have encountered the same problem. How did you solve it，thanks @jiangjiaxiaotianxia
